share/completions/protontricks-launch.fish

```diff
@@ -0,0 +1,15 @@
+function __fish_protontricks__complete_appid
+    protontricks -l |
+        string match --regex '.*\(\d+\)' |
+        string replace --regex '(.*) \((\d+)\)' '$2\t$1'
+end
+
+complete -c protontricks-launch -f -s h -l help -d 'show this help message and exit'
+complete -c protontricks-launch -l no-term -d 'Program was launched from desktop and no user-visible terminal is available. Error will be shown in a dialog instead of being printed.'
+complete -c protontricks-launch -s v -l verbose -d 'Increase log verbosity. Can be supplied twice for maximum verbosity.'
+complete -c protontricks-launch -l no-runtime -d 'Disable Steam Runtime'
+complete -c protontricks-launch -l no-bwrap -d 'Disable bwrap containerization when using Steam Runtime'
+complete -c protontricks-launch -l background-wineserver -d 'Launch a background wineserver process to improve Wine command startup time. Disabled by default, as it can cause problems with some graphical applications.'
```

Comment:
can you try to shorten the descriptions a bit (roughly 60 columns would be perfect)
looks good otherwise

Reply:
I've revised the descriptions to fit around 60 columns (I measured the description texts themselves, not the whole line, let me know if you meant the other way around).

I've also simplified some of them/made them more sensible (like "Show help message and exit" instead of "show this program's help message and exit"). Let me know if this is okay or if it's deviating too much from the original help text, `--cwd-app` is an example where I simplified it quite a bit to shorten it, but I think the description is still clear.